website/templates/emails/notify_base.mako

```diff
@@ -43,7 +46,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="http://osf.io/settings/notifications/">user settings</a>.</p>
+                            <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href=${settings.DOMAIN + "settings"}>user settings</a>. </p>
```

Comment:
I think this should link to `settings/notifications/` to take the user directly to where they can change their email preferences.